src/main/java/seedu/address/model/tag/TutorialTag.java

```diff
@@ -13,4 +13,8 @@ public class TutorialTag extends Tag {
     public TutorialTag(String tagName, TagStatus tagStatus) {
         super(tagName, tagStatus);
     }
+
+    public boolean isSameTutorialTag(TutorialTag tutorialTag) {
+        return this.equals(tutorialTag);
+    }
```

Comment:
I think it's bette to compare the name here instead of calling the equals method of the Tag class.
Eventually need to update the Tag.equals() to compare not only the tag names but also the status and/or the tag type (using polymorphism).

Reply:
Sure sure, we can just opt to override the equals method